client/components/Header/style.css

```diff
@@ -0,0 +1,20 @@
+.header {
+	display: flex;
+	margin-bottom: 3em;
+}
+
+a {
```

Comment:
we should either move this somewhere where we keep global rules for `a` or use class name and import into this component